src/code/templates/genome-playground.js

```diff
@@ -20,8 +26,19 @@ export default class GenomeContainer extends Component {
       onSexChange(0, newSex);
     };
 
-    const handleAdvanceChallenge = function() {
-      onCompleteChallenge();
+    const handleSubmit = function() {
+      let child = new BioLogica.Organism(BioLogica.Species.Drake, drakes[0].alleleString, drakes[0].sex),
+          childImage = child.getImageName(),
```

Comment:
Instead of creating a new child here, since it's just a clone of the main drake, couldn't you just do `let childImage = drake.getImageName()`?

(You'd have to rename the clashing `let drake` below, though.)